cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -1017,7 +1017,7 @@ predicate localInstructionFlow(Instruction e1, Instruction e2) {
  * local (intra-procedural) steps.
  */
 pragma[inline]
-predicate localExprFlow(Expr e1, Expr e2) { localFlow(exprNode(e1), exprNode(e2)) }
+deprecated predicate localExprFlow(Expr e1, Expr e2) { localFlow(exprNode(e1), exprNode(e2)) }
```

Comment:
This predicate exists for consistency with the data-flow libraries of other languages. Example: https://github.com/github/codeql/blob/b45f56ac08dd60cf53b2a873b74eb6ec44ef20e7/java/ql/lib/semmle/code/java/dataflow/internal/DataFlowUtil.qll#L80-L85